packages/headless/src/features/generated-answer/generated-answer-slice.test.ts

```diff
@@ -534,4 +536,23 @@ describe('generated answer slice', () => {
       expect(finalState.cannotAnswer).toEqual(false);
     });
   });
+
+  it('#setAnswerApiQueryParams should set the answerApiQueryParams to the new value in the state', () => {
+    const newAnswerApiQueryParams: Partial<SearchRequest> = {
```

Comment:
See why do we use here a Partial<SearchRequest> instead of using a SearchAPIState or the other type that was defined else where?